src/components/DiffView/styles.module.scss

```diff
@@ -41,3 +41,12 @@
   line-height: 1;
   text-align: center;
 }
+
+:global {
```

Comment:
Sorry that I didn't have time to review this patch. I think we should avoid `:global` (there really [isn't a strong use case for it](https://github.com/gajus/react-css-modules#global-css)) and I also see other issues with this patch. I filed https://github.com/mozilla/addons-code-manager/issues/346 (low priority) as a follow-up.